docs/minikube_start.md

```diff
@@ -21,7 +21,7 @@ minikube start
       --insecure-registry=[]: Insecure Docker registries to pass to the Docker daemon
       --iso-url="https://storage.googleapis.com/minikube/minikube-0.5.iso": Location of the minikube iso
       --memory=1024: Amount of RAM allocated to the minikube VM
-      --vm-driver="virtualbox": VM driver is one of: [virtualbox vmwarefusion kvm xhyve]
```

Comment:
Changed to build tags with new `gendocs` build tag so we get proper output on each OS build, but generated docs will show all supported drivers. How does that sound?